AirCasting/CreateSessionViews/ChooseSessionType/ChooseSessionTypeView.swift

```diff
@@ -34,238 +34,247 @@ struct ChooseSessionTypeView: View {
-                                    viewModel.startSync
-                                }, set: { new in
-                                    viewModel.setStartSync(using: new)
-                                })) {
-                                    CreatingSessionFlowRootView {
-                                        SDSyncRootView(creatingSessionFlowContinues: .init(get: {
-                                            viewModel.startSync
-                                        }, set: { new in
-                                            viewModel.setStartSync(using: new)
-                                        }))
-                                    }
-                                }
-                            EmptyView()
-                                .fullScreenCover(isPresented: .init(get: {
-                                    viewModel.isSearchAndFollowLinkActive
-                                }, set: { new in
-                                    viewModel.setSearchAndFollow(using: new)
-                                })) {
-                                    CreatingSessionFlowRootView {
-                                        SearchView(isSearchAndFollowLinkActive: .init(get: {
-                                            viewModel.isSearchAndFollowLinkActive
-                                        }, set: { new in
-                                            viewModel.setSearchAndFollow(using: new)
-                                        }))
-                                    }
+                            }
+                        EmptyView()
+                            .fullScreenCover(isPresented: .init(get: {
+                                viewModel.isSearchAndFollowLinkActive
+                            }, set: { new in
+                                viewModel.setSearchAndFollow(using: new)
+                            })) {
+                                CreatingSessionFlowRootView {
+                                    SearchView(isSearchAndFollowLinkActive: .init(get: {
+                                        viewModel.isSearchAndFollowLinkActive
+                                    }, set: { new in
+                                        viewModel.setSearchAndFollow(using: new)
+                                    }))
                                 }
-                        }
-                    )
-            }
-            .onAppear { defineNextMove() }
-            .onChange(of: tabSelection.selection, perform: { _ in defineNextMove() })
-            .environmentObject(viewModel.passSessionContext)
+                            }
+                    }
+                )
+                .onAppear { defineNextMove() }
+                .onChange(of: tabSelection.selection, perform: { _ in defineNextMove() })
+                .environmentObject(viewModel.passSessionContext)
         }
     }
     
-    
+    @State private var labelHeight = CGFloat.zero
```

Comment:
Maybe we could put this state next to the others variables, what do you think?

Reply:
I also changed the name to `buttonHeight` to be more descriptive